ageteller/src/components/AgeStats.js

```diff
@@ -1,20 +1,24 @@
 import React, { Component } from 'react';
+import moment from 'moment';
 import partyPopper from '../assets/party-popper.jpg';
 
 class AgeStats extends Component {
+
   timeSince(date) {
-    let today = new Date().getTime();
-    let other_date = new Date(date).getTime();
-    let difference = Math.abs(today - other_date);
+        var a = moment(new Date().getTime());
```

Comment:
Why not `const` for today, other_date, difference, years and months? Although, we will ned to use `let` for days, since we change its value.